mantidproject/mantid
# Figure Options Legend Tab

**Description of work.**
The figure options now has a legend tab so that the legend properties can be changed. 

**To test:**
Plot a figure, open figure options, there is now a Legend tab. Check that all of the different options apply to the legend correctly, and carrying out other operations on the graph maintains the legend properties.

Fixes #26596 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
